huawei_solar_battery_optimization.yaml

```diff
@@ -955,7 +972,7 @@ automation:
                         state: 'on'
                       - condition: template
                         value_template: >
-                          {{ states(inverter_entity) != "Limited to 0.0%" }}
+                          {{ states(inverter_entity) != "Limited to 0.0%" }} #Forslag: Blot kald det for "Limit Export"
```

Comment:
parameteren der sættes er "Limited to 0.0%" - så kan ikke ændre denne tekst.... men hvis vi alligevel ændrer væk fra % og over på W så skal denne alligevel ændres.